frontend/elements/src/core/share-jig/option.ts

```diff
@@ -42,12 +44,26 @@ export class _ extends LitElement {
     @property()
     kind: Kind = "students";
 
+    @property({type: Boolean})
+    urlKindClicked: boolean = false;
+
+    handleTextClick() {
+        if (this.kind !== "copy" || this.urlKindClicked) return;
+        this.urlKindClicked = !this.urlKindClicked;
+        setTimeout(() => this.urlKindClicked = false, 3000);
+    }
+
+    getStringLabelText() {
```

Comment:
Do this in Rust, the copy logic lives there, may as well do this here too.

Also, slot in the copy text